charts/rd-professional-api/values.yaml

```diff
@@ -0,0 +1,29 @@
+rd-professional-api:
+  applicationPort: 8090
+  environment:
+    POSTGRES_HOST: ${SERVICE_NAME}-postgresql  # do not modify
+    POSTGRES_SSL_MODE: disable
+    POSTGRES_USER: dbrefdata
+    POSTGRES_PASSWORD: dbrefdata
+    JSON_CONSOLE_PRETTY_PRINT:
+    ROOT_APPENDER:
+    ROOT_LOGGING_LEVEL:
+    REFORM_SERVICE_TYPE:
+    REFORM_SERVICE_NAME:
+    REFORM_TEAM:
+    REFORM_ENVIRONMENT:
+    LOGBACK_DATE_FORMAT:
+    LOGBACK_REQUIRE_THREAD:
+    LOGBACK_REQUIRE_ALERT_LEVEL: false
+    LOGBACK_REQUIRE_ERROR_CODE: false
+    S2S_URL: 'http://service-auth-provider-api:8080'
+  # Don't modify below here
+  image: ${IMAGE_NAME}
+  ingressHost: ${SERVICE_FQDN}
+
+postgresql:
+  postgresqlUsername: dbrefdata
+  postgresqlPassword: dbrefdata
+  postgresqlDatabase: hmcts
+  persistence:
+    enabled: true
```

Comment:
Do u need persistence enabled? It should have been by mistake. This is the reason why we recommend avoiding usage of postgresql chart and use the one with java as the java one has sensible defaults.

Reply:
Done, and thank you very much